index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang='cs'>
+	<head>
+    	<title>index</title>
+    	<meta charset='utf-8'>
+    	<meta name='description' content=''>
+    	<meta name='keywords' content=''>
+    	<meta name='author' content=''>
+    	<meta name='robots' content='all'>
+    	<!-- <meta http-equiv='X-UA-Compatible' content='IE=edge'> -->
+    	<link href='/favicon.png' rel='shortcut icon' type='image/png'>
+    	<meta name="viewport" content="width=device-width, initial-scale=1">                                                            
+    	<link rel="stylesheet" href="_bootstrap/bootstrap.min.css" type="text/css">                   
+    	<script src="_bootstrap/jquery.min.js"></script>               
+    	<script src="_bootstrap/bootstrap.min.js"></script>
+    	<link rel="stylesheet" href="main.css" type="text/css">
+  	</head>
+  	<body>
+    	<div class="jumbotron">
+      		<div class="header">
+        		<h1>Wcook</h1>
+        		<p>We love the food, it's a passion!!!</p>
+      		</div>
+      		<a class="button najitrecept" href="selection.html">
+      		Najít<br>recept
+      		</a>
+    	</div>
+    	<div class=><br>
+  			<div id="myCarousel" class="carousel slide" data-ride="carousel">
+    			<ol class="carousel-indicators">
+      				<li data-target="#myCarousel" data-slide-to="0" class="active"></li>
+      				<li data-target="#myCarousel" data-slide-to="1"></li>
+      				<li data-target="#myCarousel" data-slide-to="2"></li>
+    			</ol>
+
+    		<div class="carousel-inner" role="listbox">
+      			<div class="item active">
+        			<img src="upoutavky/certuvkotlik.jpg" class="image">
+      			</div>
+      			<div class="item">
+        			<img src="upoutavky/cesar.jpg" class="image">
+      			</div>
+      			<div class="item">
+        			<img src="upoutavky/spagety.jpg" class="image">
+      			</div>
+    				<a class="left carousel-control" href="#myCarousel" role="button" data-slide="prev">
+      					<span class="glyphicon glyphicon-chevron-left" aria-hidden="true"></span>
+      					<span class="sr-only">Previous</span>
+    				</a>
+    				<a class="right carousel-control" href="#myCarousel" role="button" data-slide="next">
+      					<span class="glyphicon glyphicon-chevron-right" aria-hidden="true"></span>
+      					<span class="sr-only">Next</span>
+    				</a>
+  				</div>
+			</div>
+		</div>
+    <div class="footer">
+    Copyright © Wcook.4fan.cz
+    </div>
+  <div ><div class="dulezite"><div class="eq"><i>„Zamilovat se do sebe - to je románek na celý život.“ </i>Oscar Wilde</div></div></div></body>
```

Comment:
Footer is the last element on web. Put this quote in another place, probably in footer.